README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/robertmain/node-pureimage.svg?branch=master)](https://travis-ci.org/robertmain/node-pureimage)
```

Comment:
This is going to need changed when the PR is merged of course to point at your repo